src/Config.php

```diff
@@ -26,6 +26,11 @@ public static function getInstance(): Config
     private function __construct()
     {
         $this->config = include __DIR__ . '/../config.php';
+
+        if (file_exists(__DIR__ . '/../config.local.php')) {
+            $localConfig = include __DIR__ . '/../config.local.php';
+            $this->config = array_merge_recursive($this->config, $localConfig);
```

Comment:
If elements will be in both arrays we will get array of values instead of value.